numpyro/hmc_util.py

```diff
@@ -590,6 +595,16 @@ def initialize_model(rng, model, *model_args, **model_kwargs):
     model_trace = trace(model).get_trace(*model_args, **model_kwargs)
     sample_sites = {k: v for k, v in model_trace.items() if v['type'] == 'sample' and not v['is_observed']}
     inv_transforms = {k: biject_to(v['fn'].support) for k, v in sample_sites.items()}
-    init_params = constrain_fn(inv_transforms, {k: v['value'] for k, v in sample_sites.items()}, invert=True)
+    prior_samples = constrain_fn(inv_transforms,
+                                 {k: v['value'] for k, v in sample_sites.items()}, invert=True)
```

Comment:
While you address the above `rng` issue, could you change the name of `prior_samples` too. For example, `prior_params` or `prior_unconstrained...` will be less confusing.